documentation/cloud-manager-guide/secure-data-connectors/add-data-connector.md

```diff
@@ -0,0 +1,72 @@
+---
+keywords:
+title: Add a Data Connector
+description: Add a data connector
+---
+# Add a Data Connector
+
+This guide outlines the steps to add a new secure data connector in Environment Operations Center.
+
+[!info] The steps outlined in this guide assume you have already configured the OPA client to establish the WebSocket tunnel between your on-prem environment and Env Ops Center. For details on how to configure the OPA client, see the [configure an OPA client](...) guide. **(update link when available)**
+
+## Getting started
+
+Adding a new data connector in Env Ops Center is broken out into two steps: Creating the data connector by providing its basic information and registering the data connector.
+
+To add a new data connector, select **Add New Connector** from the *Secure Data Connectors* home screen.
+
+![image description](images/new-connector.png)
+
+## Add data connector info
+
+In the *New Data Connector* dialog, enter the required data connector info in the provided name and a description fields.
+
+> **RL QUESTION: waiting on confirmation re: name and description best practices**
+
+![image description](images/add-connector-info.png)
+
+Once you have completed the required fields in the *Data Connector Info* section, select **Save** to add the new connector.
+
+![image description](images/add-save.png)
+
+If the data connector is successfully created, it will appear in the list of available data connectors on the *Secure Data Connectors* home screen.
+
+![image description](images/add-new-connector.png)
+
+## Data connector registration
+
+A new data connector needs to be registered for it to become active. The status of a data connector indicates if the connector is "Active" or requires registration. A new connector that has not been registered will have an "Unregistered" status.
+
+![image description](images/add-unregistered.png)
+
+To register a connector, select the connector name to open the connector details.
+
+![image description](images/add-select-name.png)
```

Comment:
I think text lines 42 and 46 could be combined and then screen shots on line 44 and 48 could just highlight both options. Save scrolling since it's pretty easy functionality to follow.